RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -482,3 +484,19 @@ extension VoiceBroadcastPlaybackViewModel: VoiceMessageAudioPlayerDelegate {
         stopIfVoiceBroadcastOver()
     }
 }
+
+// MARK: - NowPlayingInfoProvider
+
+extension VoiceBroadcastPlaybackViewModel: VoiceMessageNowPlayingInfoProvider {
+    func updatePlayingInfoCenter(forPlayer player: VoiceMessageAudioPlayer) {
+        guard audioPlayer != nil, audioPlayer === player else {
+            return
+        }
+        
+        let nowPlayingInfoCenter = MPNowPlayingInfoCenter.default()
+        nowPlayingInfoCenter.nowPlayingInfo = [MPMediaItemPropertyTitle: VectorL10n.voiceBroadcastPlaybackLockScreenPlaceholder,
+                                    MPMediaItemPropertyPlaybackDuration: (state.playingState.duration / 1000.0) as Any,
+                            MPNowPlayingInfoPropertyElapsedPlaybackTime: (state.bindings.progress / 1000.0) as Any,
+                                   MPNowPlayingInfoPropertyPlaybackRate: state.playbackState == .playing ? 1 : 0]
```

Comment:
This is just a thought but perhaps we should let the `VoiceMessageMediaServiceProvider` deal with updating playback values from its `currentlyPlayingAudioPlayer` and only update the title from the VoiceMessageNowPlayingInfoProvider`. It would also work better with the "Provider" name in which it "provides" a title instead of directly manipulting the InfoCenter

Reply:
In the case of a voice broadcast we should not use values from its `currentlyPlayingAudioPlayer`. A voice broadcast is composed by multiple chunks. Each chunk is a voice message. So `currentlyPlayingAudioPlayer` has values for one chunk only, the one that is being played.

Reply:
Oh I understand, sorry for the noise then 👍